lib/api/kuzzle.js

```diff
@@ -110,9 +133,9 @@ function Kuzzle () {
     return this.internalEngine.init()
       .then(() => this.internalEngine.bootstrap.all())
       .then(() => this.pluginsManager.packages.bootstrap())
-      .then(() => this.validation.init())
```

Comment:
validation.init should be done before the the pluginManager.init as plugins can add types and pre-defined types should be loaded first.